### ISSUE TITLE
Build error, cannot find file "translation_ca.qm"

### ISSUE THREAD

**Issue by paolopugay** [raspberrypi/noobs#448]
How was this issue encountered?
1. Clone and download the project files (.zip) - noobs-master.zip (10-26-2017 - 5:33PM +4GMT)
2. Extract to a (new) folder
3. Open a terminal and type/perform the following commands:
a."sudo apt-get install build-essential rsync texinfo libncurses-dev whois unzip bc qt4-linguist-tools"
b."./BUILDME.sh"

NOTES:
A. that translation files (*.ts) are inside the "recovery" folder, and no ALL file are not modified.
B. encountered using VMWARE Workstation 12, having LINUX LITE 3.6 - xenial
C. Might be similar to #429, but I decided to open a new issue (rather than re-open) as the version/date of NOOBS file was not mentioned 
D. see attached screenshot
![2017-10-27-185219_1360x768_scrot](https://user-images.githubusercontent.com/17675380/32115706-e56f6516-bb58-11e7-9f01-53b13433b70c.png)

**Comment by XECDesign** [comment]
No problems using my build environment, even when downloading the .zip (which I suspected would be the problem). Downloading Linux Lite to test in a VM.

**Comment by XECDesign** [comment]
![screenshot_pixel_x86_2017-11-01_19 12 25](https://user-images.githubusercontent.com/1716509/32292654-b11601ec-bf38-11e7-8c51-012668b27d1c.png)

Still fine. Have I missed something?

**Comment by paolopugay** [comment]
Hi XECDesign,

Thanks a lot for the assistance. I will try to re-create your exact same procedure in my VM.
Also, my laptop is not that fast, I will try to provide a feedback as soon as I can.

**Comment by paolopugay** [comment]
Hi XECDesign,

The same issue(s) was still **encountered**. However, this time I did manage to find a workaround to have an output. The only thing missing is the language translations. 

![img_4570](https://user-images.githubusercontent.com/17675380/32303457-25d0ad90-bf82-11e7-9af5-e048c552e484.JPG)

Maybe you can just send me all the ***.qm files**?
Anyway, I can still use this in this condition.

Also, here's the history of the build:

1. Error was encountered
![2017-11-02-021516_1360x768_scrot](https://user-images.githubusercontent.com/17675380/32303531-948c4eb0-bf82-11e7-913a-9b8b394439f0.png)

2. Updating the **icons.qrc** (changing **.qm** to **.ts**)
![2017-11-02-021951_1360x768_scrot](https://user-images.githubusercontent.com/17675380/32303592-eb5028b6-bf82-11e7-99e4-6d261fb86878.png)
![2017-11-02-022128_1360x768_scrot](https://user-images.githubusercontent.com/17675380/32303598-f086fd50-bf82-11e7-8000-557d393f7913.png)

3. re-run .**/BUILDME.sh**. In here, it displayed that the ***.qm** files is missing but it can continue
![2017-11-02-022813_1360x768_scrot](https://user-images.githubusercontent.com/17675380/32303651-5523009c-bf83-11e7-8b20-af4dd1bd4a03.png)

4. Build completed
![2017-11-02-032610_1360x768_scrot](https://user-images.githubusercontent.com/17675380/32303667-6eab2cce-bf83-11e7-8e39-0dc10119e5d0.png)

5. Bash history
![2017-11-02-032858_1360x768_scrot](https://user-images.githubusercontent.com/17675380/32303672-7c4bba88-bf83-11e7-96e6-b27b6d5d0410.png)

**Comment by XECDesign** [comment]
Undo your hacky workaround and go back to the failing state.
What do you have in you `buildroot/output/build/recovery-1.0` directory?
Is the translation_ca.qm file in there?
What happens if you `cd` there and run `./updateqm.sh`?

**Comment by paolopugay** [comment]
Hi XECDesign,

I already did a successful build without any issues.

![img_4573](https://user-images.githubusercontent.com/17675380/32357018-701517d0-c052-11e7-914a-996460f09e4d.JPG)

Thanks for pointing out that I need to check the **./updateqm.sh**.

To answer your questions: 

1. By reverting back in using the original **icons.qrc** file, the ***.qm** files were not created in the **buildroot/output/build/recovery-1.0** directory

2. By running the **./updateqm.sh**, errors occurred:

Please see the details in terminal as the file manager was not updated/refreshed

![2017-11-03-033320_1360x768_scrot](https://user-images.githubusercontent.com/17675380/32357217-2b17e7b4-c054-11e7-84bb-96acb4aed1f3.png)

![2017-11-03-034127_1360x768_scrot](https://user-images.githubusercontent.com/17675380/32357164-a37526c8-c053-11e7-9f73-64c5d2c4a4c3.png)

By installing the **qt4-dev-tools**, solves the issues

![2017-11-03-042135_1360x768_scrot](https://user-images.githubusercontent.com/17675380/32357280-d71506d2-c054-11e7-9c3d-19273286143b.png)

![2017-11-03-043345_1360x768_scrot](https://user-images.githubusercontent.com/17675380/32357305-0763443e-c055-11e7-8251-d1fa3ec60d1f.png)

Again, thanks a lot for the assistance :)

**Comment by XECDesign** [comment]
I don't think you ran `sudo apt-get install build-essential rsync texinfo libncurses-dev whois unzip bc qt4-linguist-tools`. lupdate comes from qt4-linguist-tools and your screenshot shows it being installed (which wouldn't be the case if it was already installed earlier).
